spec/lib/modules/embeddings/vector_representations/vector_rep_shared_examples.rb

```diff
@@ -25,7 +25,7 @@
         )
       stub_vector_mapping(text, @expected_embedding)
 
-      vector_rep.generate_topic_representation_from(topic)
+      vector_rep.generate_representation_from(topic)
```

Comment:
also add a minimal test around here for a post embedding? at least code will get covered.